client/my-sites/theme/bundle-settings.tsx

```diff
@@ -0,0 +1,22 @@
+interface BundleSettings {
+	[ key: string ]: {
+		name: string;
+		BadgeIcon: React.ComponentType;
+	};
+}
+
+const bundleSettings: BundleSettings = {
+	'woo-on-plans': {
```

Comment:
I was planning to change the `'woo-on-plans'` to a constant. But it appears in many places, so I didn't want to risk an accidental issue for now.